### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by SiposZoltan**

### CODE REVIEW

**Comment by SiposZoltan** [comment]
This pull request is between multiple me's

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# Git_tech_cloud
```
Comment (cafeaua) [inline README.md]: Comment number one 17 54

**Comment by SiposZoltan** [comment]
deleted this branch too 19 47

**Review by cafeaua — COMMENTED** [review]
okidoki 20 00

**Pull request merged**